integrationExamples/gpt/pbjs_ucfunnel_gpt.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <!-- Prebid Config Section START -->
+  <!-- Make sure this is inserted before your GPT tag -->
+  <script>
+    var PREBID_TIMEOUT = 1000;
+
+    var adUnits = [{
+    code: 'div-gpt-ad-1495530934718-0',
+    sizes: [[300, 250]],
+    bids: [{
+        bidder: 'ucfunnel',
+        params: {
+          adid: "test-ad-83444226E44368D1E32E49EEBE6D29",        //String - required
+          width: 300,
+          height: 250
+        }
+      }
+    ]
+  }];
+
+    var pbjs = pbjs || {};
+    pbjs.que = pbjs.que || [];
+
+  </script>
+  <!-- Prebid Config Section END -->
+
+  <!-- Prebid Boilerplate Section START. No Need to Edit. -->
+  <script type="text/javascript" src="//cdn.aralego.net/pbjs/v1/prebid.js" async></script>

```

Comment:
I do not see ad loading on this page but getting bid responses.

I think you should point this to `/build/dev/prebid.js`
I don't know how frequently you update your cdn prebid.js but using build version will always be latest.